src/test/java/com/lebediev/movieland/dao/jdbc/integrationtests/JdbcMovieDaoITest.java

```diff
@@ -85,5 +86,54 @@ public void testGetMovieById() {
         assertNotEquals(actual, 0);
     }
 
+    @Test
+    public void testAdd() {
+        MovieDtoForUpdate movie = new MovieDtoForUpdate();
+        movie.setNameNative("ITest movie");
+        movie.setYearOfRelease(1900);
+        movie.setCountries(new int[]{1, 2, 3});
+        movie.setGenres(new int[]{4, 5});
+        jdbcMovieDao.add(movie);
+        int result = jdbcTemplate.queryForObject("SELECT COUNT(*) FROM movie WHERE nameNative = 'ITest movie' AND yearOfRelease = 1900", Integer.class);
+        assertEquals(result, 1);
+        int id = jdbcTemplate.queryForObject("SELECT id FROM movie WHERE nameNative = 'ITest movie' AND yearOfRelease = 1900", Integer.class);
+        result = jdbcTemplate.queryForObject("SELECT COUNT(*) FROM movie2country WHERE movieId = " + id, Integer.class);
+        assertEquals(result, 3);
+        result = jdbcTemplate.queryForObject("SELECT COUNT(*) FROM movie2genre WHERE movieId = " + id, Integer.class);
+        assertEquals(result, 2);
+        jdbcTemplate.update("DELETE FROM movie WHERE nameNative = 'ITest movie' AND yearOfRelease = 1900");
+        jdbcTemplate.update("DELETE FROM movie2country WHERE movieId = " + id);
+        jdbcTemplate.update("DELETE FROM movie2genre WHERE movieId = " + id);
+    }
+
+    @Test
+    public void testUpdate() {
+        MovieDtoForUpdate actual = new MovieDtoForUpdate();
+        actual.setNameNative("before update");
+        actual.setYearOfRelease(2000);
+        actual.setCountries(new int[]{1, 2, 3});
+        actual.setGenres(new int[]{4, 5});
+        jdbcMovieDao.add(actual);
```

Comment:
i think that this method returns id of added movie, so below you don't need after update method select by nativeName: use returned id for further checks

Reply:
it returns id inside, not visible outside